File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        int index = 0;
        for (int i = 0; i < nums.length; i++) {
            index++;
            for (int j = index; j < nums.length; j++) {
                if (nums[i] == nums[j]) {
                    return true;
                }
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if not nums: 
            return 0
            
        nums.sort()
        count = 1
        longest = 0
        for i in range(len(nums) - 1):
            if (nums[i + 1] - nums[i] == 1):
                count += 1
            elif(nums[i + 1] == nums[i]):
                count += 0
            else:
                longest = max(count, longest)
                count = 1
        return max(longest, count)


            
        